### ISSUE TITLE
运行preprocess_wos.py出错

### ISSUE THREAD

**Issue by qinglongheu** [wzh9969/contrastive-htc#15]
运行preprocess_wos.py出错FileNotFoundError: [Errno 2] No such file or directory: 'Meta-data/Data.txt'， 请问这是为什么？

**Comment by wzh9969** [comment]
这是数据集的原始数据文件，请参考readme中的链接下载数据集并放置于`data`文件夹内

**Comment by qinglongheu** [comment]
这个是preprocess_wos.py中的代码， readme中的连接下载的meta_data文件夹中却是data.xlsx文件
`import os
import json
import numpy as np
from sklearn.model_selection import train_test_split
import re

"""
WoS Reference: https://github.com/kk7nc/HDLTex
"""

FILE_DIR = 'Meta-data/Data.txt'
total_len = []
np.random.seed(7)
`

**Comment by wzh9969** [comment]
用excel另存为txt文件即可
